### PR TITLE
Editorial: Fix some mathematical type issues around PlainTime.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: polyfill/lib/plaintime.mjs [hunk 1]
```diff
@@ -66,12 +66,12 @@ function TemporalTimeToString(time, precision, options = undefined) {
 
 export class PlainTime {
   constructor(isoHour = 0, isoMinute = 0, isoSecond = 0, isoMillisecond = 0, isoMicrosecond = 0, isoNanosecond = 0) {
-    isoHour = ES.ToInteger(isoHour);
-    isoMinute = ES.ToInteger(isoMinute);
-    isoSecond = ES.ToInteger(isoSecond);
-    isoMillisecond = ES.ToInteger(isoMillisecond);
-    isoMicrosecond = ES.ToInteger(isoMicrosecond);
-    isoNanosecond = ES.ToInteger(isoNanosecond);
+    isoHour = ES.ToFiniteInteger(isoHour);
```
Comment (ljharb) [inline polyfill/lib/plaintime.mjs:69]: where is this AO defined?

Comment (ptomato) [reply polyfill/lib/plaintime.mjs:69]: See https://github.com/tc39/proposal-temporal/pull/1554#discussion_r657388876

**Review by ptomato — APPROVED** [review]
:+1:

File: polyfill/lib/plaintime.mjs (hunk 1 above)
Comment (ljharb) [reply polyfill/lib/plaintime.mjs:69]: why add it to the polyfill before there's spec text for it?

**Pull request merged**